bin/travis_mac_setup.sh

```diff
@@ -0,0 +1,8 @@
+rew update
```

Comment:
Also adding a proper `#!` header would make it usable as a stand-alone script for installing tools.